package/src/module.js

```diff
@@ -181,7 +187,7 @@ function getTextfromList(el) {
       result.push(child)
     } else if (child.nodeName === 'SPAN') {
       child.textContent = child.textContent.replace(
-        /(^(\W)(?=\s)*)|(o\s)(?!\w)/gm,
+        /(^(\W)(?=\s)*)|(o\s)(?!\w)|[0-9]\)/gm,
```

Comment:
needed to add detection of `1)` here to ignore that text. There is also a style attribute `style="mso-list:Ignore"` that maybe we could use here instead?